tests/pipelines/ledits_pp/test_ledits_pp_stable_diffusion.py

```diff
@@ -0,0 +1,237 @@
+# coding=utf-8
```

Comment:
Let's try to make sure that the tests don't add a big overhead in terms of compute time (both for fast and slow tests).